src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -659,14 +661,15 @@ def create_python_udf_or_sp(
     )
 
     create_query = f"""
-CREATE {"OR REPLACE " if replace else ""}
+CREATE{" OR REPLACE " if replace else ""}
```

Comment:
This is an unrelated change, it just removes a space from the generated SQL command